examples/2016-10-31/api_aws_iam_auth/template.yaml

```diff
@@ -8,7 +8,7 @@ Resources:
       StageName: Prod
       Auth:
         DefaultAuthorizer: AWS_IAM
-        InvokeRole: CALLER_CREDENTIALS
+        InvokeRole: CALLER_CREDENTIALS # default, set to NONE or null to not require credentials
```

Comment:
This isn't accurate. IAM credentials are still required, they just aren't used as the invocation role for the Lambda Function. Setting to NONE will use the Lambda Function's configured role; setting to an different IAM role will invoke using that role.